Infra/createLambdaHandler.js

```diff
@@ -2,21 +2,20 @@ const dependencies = {
   logger: require('./logger')
 }
 
-module.exports = function createLambdaHandler (useCase, injection) {
-  return function (event, context, callback) {
+module.exports = function createLambdaHandler (handler, injection) {
+  return async function lambdaHandler (event, context, callback) {
```

Comment:
I'm not sure, but now with an `async` function as a lambdaHandler we can skip the callback and return the function value as the result (changing `return callback(null, result)` to just `return result`) and a rejection as an error (`return Promise.reject(error)` instead of `return callback(error)`).